rpm/generic/zfs.spec.in

```diff
@@ -268,8 +271,11 @@ Requires:       libnvpair1 = %{version}
 Requires:       libffi
 Requires:       python%{__python_pkg_version}
 Requires:       %{__python_cffi_pkg}
+Requires:       %{__python_setuptools_pkg}
```

Comment:
Why do we need setuptools at runtime? Are we generating any scripts dynamically that pull in setuptools to function? I didn't see any when looking at the pyzfs code.

Reply:
I wasn't sure, so I added it.  I'll drop this and refresh the PR.